### DISCUSSION TITLE
小求助

### DISCUSSION THREAD

**Discussion by shiepo** [BYJRK/DotNet-Discussions#15] [Q&A] [answered]
请问一下大佬，这个功能是插件达到的么
![微信图片_20230203223651](https://user-images.githubusercontent.com/52399003/216630236-22b0a024-b134-4dbb-9a6d-b4d0bab956d2.png)

**Comment by yoi102** [comment]
是不是安装了Resharper？

**Comment by yoi102** [comment]
企业版VS也会有的

**Comment by BYJRK** [comment, marked as answer]
这个是vs自带的功能，在选项，文本编辑器，C#，高级，内联提示

**Reply by shiepo** [reply]
已实现，感谢大佬
